app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
+    include FormConfigurable
+    include Oauthable
+    valid_oauth_providers :'lifx'
+    
+    cannot_be_scheduled!
+    cannot_create_events!
+
+    description <<-MD
+      Pulses your your [LIFX lights](http://lifx.com/) between their current color and a color you specify.
+      
+      To be able to use this Agent you need to authenticate with LIFX in the [Services](/services) section first.
+
+      Read more about the [LIFX HTTP API](http://api.developer.lifx.com/) 
```

Comment:
Would it make sense to include the documentation from http://api.developer.lifx.com/docs/pulse-effect here?